Services/UserService/Db/IDbService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Common.Enums;
+using Common.Models;
+using Common.Models.User;
+
+namespace UserService.Service
```

Comment:
We are lacking all the rest of CRUD operations. There is no Create, Update or Delete User.